core/sdk-core/src/test/java/software/amazon/awssdk/core/internal/http/timers/HttpClientApiCallTimeoutTest.java

```diff
@@ -60,7 +60,6 @@ public class HttpClientApiCallTimeoutTest {
     @Before
     public void setup() {
         httpClient = testClientBuilder()
-            .retryPolicy(RetryPolicy.none())
```

Comment:
Why we need to remove this now  ?

Reply:
Well, the class is deprecated so I figured we should also try to remove its uses, technically we don't have to do it now.

Reply:
hmm... I might be over thinking but I was thinking of a case where due to some reason we want to modify older RetryPolicy  (since they are not removed but just deprecated) , we might loose test cases for them

Reply:
Sorry, I missed this comment, yeah, I think that's a fair concern, I will bring this up during review.